docs/macros.md

```diff
@@ -991,24 +994,28 @@ Generates SQL to build an Extended Tracking Satellite table using the provided p
     === "Single-Source"
 
         ```sql
-        WITH 
-        satellite_SATELLITE_1_from_PRIMED_STAGE AS (
+        WITH satellite_SATELLITE_1_from_STG_CUSTOMER AS (
```

Comment:
These should be easily readable names. If this is copy and pasted from actual SQL that's great, but some minor modifications should be made to make the names of things more realistic.

Reply:
Renamed the satellite CTEs to "satellite_a", "satellite_b" etc. 

Indeed. These CTEs need a better name "formula".

Reply:
Let's remove the tip entirely.